build-tools/packages/build-cli/src/library/typescriptApi.ts

```diff
@@ -43,30 +45,44 @@ function isTypeExport(_decl: ExportedDeclarations): boolean {
 	return false;
 }
 
+interface ApiSupportTagInfo {
+	releaseTag: ReleaseTag;
+	isLegacy: boolean;
```

Comment:
I'd love to have some clear documentation for what legacy actually means. Assuming we have some, link it from here. Otherwise at least provide a minimal doc comment